tests/README.md

```diff
@@ -28,7 +28,7 @@ Go to your "everest-core/**tests**" folder and execute *pytest*:
 
 ```bash
 cd ~/checkout/everest-workspace/everest-core/tests
-pytest-3 -s ./core_tests/startup_tests.py --path .. --junitxml=results.xml
+pytest --everest-prefix ../build/dist ./core_tests/startup_tests.py
```

Comment:
```suggestion
pytest --everest-prefix ../build/dist core_tests/*.py framework_tests/*.py
```

Reply:
This is what we are doing in https://github.com/EVerest/everest-core/blob/main/tests/run-testing.sh and https://github.com/EVerest/everest-core/blob/main/.ci/e2e/scripts/tests.sh